packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -141,24 +140,13 @@ const ReleasesGrid = ({ sectionTitle, releases = [], isError = false }: Releases
                   {name}
                 </Typography>
                 <Typography variant="pi" textColor="neutral600">
-                  {IsSchedulingEnabled ? (
-                    scheduledAt ? (
-                      <CapitalizeRelativeTime timestamp={new Date(scheduledAt)} />
-                    ) : (
-                      formatMessage({
-                        id: 'content-releases.pages.Releases.not-scheduled',
-                        defaultMessage: 'Not scheduled',
-                      })
-                    )
+                  {scheduledAt ? (
```

Comment:
could `content-releases.page.Releases.release-item.entries` be deleted from the translation file?

Reply:
Good catch, thank you!